src/date_type.h

```diff
@@ -104,13 +101,13 @@ static const Year MAX_YEAR  = 5000000;
  * @see ConvertDateToYMD(), ConvertYMDToDate()
  */
 struct YearMonthDay {
-	Year  year;   ///< Year (0...)
-	Month month;  ///< Month (0..11)
-	Day   day;    ///< Day (1..31)
+	TimerGameCalendar::Year  year;   ///< Year (0...)
+	TimerGameCalendar::Month month;  ///< Month (0..11)
+	TimerGameCalendar::Day   day;    ///< Day (1..31)
 };
 
-static const Year  INVALID_YEAR  = -1; ///< Representation of an invalid year
-static const Date  INVALID_DATE  = -1; ///< Representation of an invalid date
+static const TimerGameCalendar::Year  INVALID_YEAR  = -1; ///< Representation of an invalid year
+static const TimerGameCalendar::Date  INVALID_DATE  = -1; ///< Representation of an invalid date
```

Comment:
So many silly alignments that now no longer make sense. Nothing you have to fix, but just to show how silly those space-alignments actually are.